app/mailers/invoice_mailer.rb

```diff
@@ -12,9 +12,9 @@ def invoice
 
```

Comment:
Can you update the above line with,
```suggestion
@invoice_url = "#{ENV.fetch("APP_BASE_URL", "https://app.getmiru.com")}/invoices/#{@invoice.id}/view"
```